docs/operator-manual/app-any-namespace.md

```diff
@@ -71,6 +71,8 @@ We supply a `ClusterRole` and `ClusterRoleBinding` suitable for this purpose in
 kubectl apply -f examples/k8s-rbac/argocd-server-applications/
 ```
 
+`argocd-notifications-controller-rbac-clusterrole.yaml` and `argocd-notifications-controller-rbac-clusterrolebinding.yaml` are used to support notifications controller to notify apps in all namespaces.
```

Comment:
Can you add these two files to examples/k8s-rbac/argocd-server-applications/kustomization.yaml's `resources`?